node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/network/IPAddresses.java

```diff
@@ -56,12 +57,14 @@ default List<String> getAddresses(String hostname, IPVersion ipVersion) {
      * @throws ConvergenceException if multiple addresses are found
      */
     default Optional<Inet6Address> getIPv6Address(String hostname) {
+        CidrBlock gcpInternalBlock = CidrBlock.fromString("2600:2d00::/32");
```

Comment:
Make static?

Reply:
Done.  Unfortunately it cannot be made private.